### PR TITLE
feat(hardhat): add default hardhat test fixture

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

To run tests completely with hardhat, we need a default fixture that is used alongside deployments to do additional setup.

**Summary**

This adds a fixtures directory and a default fixture to common/hardhat. The fixture essentially mirrors the additional logic in truffle migrations beyond the core deployments.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [x]  Untested

**Issue(s)**

N/A

### CODE REVIEW

File: packages/common/src/hardhat/fixtures/default.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    await addToFinder("FinancialContractsAdmin", interfaceName.FinancialContractsAdmin);
+    await addToFinder("Voting", interfaceName.Oracle);
+    await addToFinder("Registry", interfaceName.Registry);
+    await addToFinder("Store", interfaceName.Store);
+    await addToFinder("IdentifierWhitelist", interfaceName.IdentifierWhitelist);
+    await addToFinder("AddressWhitelist", interfaceName.CollateralWhitelist);
+    await addToFinder("OptimisticOracle", interfaceName.OptimisticOracle);
+    await addToFinder("Bridge", interfaceName.Bridge);
+    await addToFinder("GenericHandler", interfaceName.GenericHandler);
```
Comment (nicholaspai) [inline packages/common/src/hardhat/fixtures/default.js:32]: Also need `FundingRateStore` right? Or are we not yet adding the Perp-related contracts to prod set up

Comment (mrice32) [reply packages/common/src/hardhat/fixtures/default.js:32]: I don't think we have a funding rate store. I think that mapping element is a leftover from before we deleted that contract.

Comment (mrice32) [reply packages/common/src/hardhat/fixtures/default.js:32]: I can remove the mapping element, though!

Comment (mrice32) [reply packages/common/src/hardhat/fixtures/default.js:32]: Done.

Comment (nicholaspai) [reply packages/common/src/hardhat/fixtures/default.js:32]: Oh you're right. Can you remove the mapping in this PR

Comment (mrice32) [reply packages/common/src/hardhat/fixtures/default.js:32]: Just did it :)

**Review by nicholaspai — APPROVED** [review]
LGTM

File: packages/common/src/hardhat/fixtures/default.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    const getDeployment = async (name) => {
+      const contract = await deployments.get(name);
+      return new web3.eth.Contract(contract.abi, contract.address);
+    };
+
+    // Setup finder.
+    const finder = await getDeployment("Finder");
+
+    const addToFinder = async (deploymentName, finderName) => {
+      const { address } = await deployments.get(deploymentName);
```
Comment (chrismaree) [inline packages/common/src/hardhat/fixtures/default.js:19]: this is cool.

**Review by chrismaree — APPROVED** [review]
LGTM! this pretty sweet

**Pull request merged**